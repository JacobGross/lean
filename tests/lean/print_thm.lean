open nat

theorem simple (a : nat) : a â‰¥ 0 :=
zero_le a

print simple

reveal simple

print simple
