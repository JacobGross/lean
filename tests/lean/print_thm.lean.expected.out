theorem simple : ∀ (a : ℕ), a ≥ 0
'simple' is still in the theorem queue, use command 'reveal simple' to access its definition.
theorem simple : ∀ (a : ℕ), a ≥ 0 :=
zero_le
